docs/code_ref/sunpy_stability.yaml

```diff
@@ -5,10 +5,10 @@ data:
   status: mature
```

Comment:
I wonder since we added a new method and will remove the old one. Maybe this should be stable?